assets/sass/_navigation.scss

```diff
@@ -84,6 +84,7 @@ Style guide: Navigation.Primary Navigation
     background-color: $background-colour;
     text-decoration: none;
     color: $non-black;
+    font-size: rem(14);
```

Comment:
This came from sketching some of the layout in code. We felt like a smaller font size on the breadcrumbs created a more distinctive hierarchy between the breadcrumbs and the surrounding elements on the page.

From the designs I've seen there are a few other components that have similar small font sizes. Maybe we need to have a broader conversation around balancing the visual design and usability of the typography?